game_eggs/rimworld/open_world/egg-rimworld-open-world.json

```diff
@@ -0,0 +1,272 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2022-12-01T15:59:26+00:00",
+    "name": "Open World",
+    "author": "kyle@williquette.us",
+    "description": "The Open World mod allows you to create and join servers to play with people all around the world, it introduces plenty of new features including trading, gifting, bartering, PvP and so much more!\r\n\r\nhttps:\/\/github.com\/TastyLollipop\/OpenWorld",
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/parkervcp\/yolks:debian": "ghcr.io\/parkervcp\/yolks:debian"
+    },
+    "file_denylist": [],
+    "startup": "DOTNET_ROOT=\/home\/container;\/home\/container\/\"Open World Server\"",
```

Comment:
This could be an egg ENV instead `DOTNET_ROOT=\/home\/container`

Reply:
Just made the change and did a fix to the max_player from being limited to 3 max to now allow unlimited players(oops)